eosdis-nasa/earthdata-pub-api
# Remove Confirmation Form From Repo

# Description

Delete remaining confirmation form artifacts from repo to avoid any confusions in the future.

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Ensure that everything works as expected and that there are no references to the confirmation form within the repo.

---
